metadata/boundaries/zip-code-areas

```diff
@@ -0,0 +1,83 @@
+# Title
+
+Utah ZIP Code Areas
+
+## ID
+
+1d4cbfba-75da-4d7c-a9bb-b32902b0af6d
+
+## Brief Summary
+
+The estimated geographic extents for all ZIP codes in Utah, excluding those that are PO Box only.
+
+## Summary
+
+This dataset includes the approximate geographic boundaries of every five-digit Zone Improvement Program (ZIP) code in Utah. ZIP codes are approximated based on the locations of known mailing addresses and are represented as polygons.
+
+## Description
+
+### What is the dataset?
+
+ZIP codes were first introduced by the United State Postal Service (USPS) in 1963 to improve mail delivery. The first three digits of the five-digit code refer to the Sectional Center Facility (SCF), or the central processing facility for a given region's mail. The last two digits provide a more specific locale that the mail will be delivered to.
+
+### What is the purpose of the dataset?
+
+This dataset was developed to provide a reference for assigning ZIP codes to geocoded features in the [Utah Roads](https://gis.utah.gov/products/sgid/transportation/road-centerlines/) and [Utah Address Points](https://gis.utah.gov/products/sgid/location/address-points/) datasets. Utah ZIP Code Areas have also been used in a variety of analyses; for example, one such analysis maps housing affordability in Utah by ZIP code. You can read about how ZIP codes were used to quantify housing data on our [blog](https://gis.utah.gov/blog/2024-05-17-utah-housing-affordability-map/).
+
+This dataset should not be used to determine the definitive zip code for addressing mail that will be sent through the US Postal Service.  The [USPS website](https://tools.usps.com/zip-code-lookup.htm) should be used for this purpose.
```

Comment:
**polish**: we should reference ZIP the same way though out this document.
**polish**: 1 space after punctuation!

```suggestion
This dataset should not be used to determine the definitive ZIP code for addressing mail that will be sent through the US Postal Service. The [USPS website](https://tools.usps.com/zip-code-lookup.htm) should be used for this purpose.
```